bandcamp_dl/__main__.py

```diff
@@ -105,13 +104,15 @@ def main():
     elif arguments.artist and arguments.track:
         urls = Bandcamp.generate_album_url(arguments.artist, arguments.track, "track")
     elif arguments.artist:
-        urls = Bandcamp.get_full_discography(arguments.artist, "music")
+        urls = Bandcamp.get_full_discography(bandcamp, arguments.artist, "music")
     else:
         urls = arguments.URL
 
     album_list = []
 
     for url in urls:
+        if "/album/" not in url and "/track/" not in url:
```

Comment:
Any specific reasoning for this?

Reply:
Ah I see this is related to #233 and just got carried over.

Reply:
In my testing the `urls` list often (always?) contains a top-level url of the artist without any albums or tracks, and is the first in the list, causing `bandcamp-dl` to immediately fail.

This could potentially be fixed in get_full_discography(), but I'm not that good at python.